docs/content/python/getting-started/application/_index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Add Oso to Your Python App
+weight: 2
+description: |
+    In this guide, we'll cover the basics of adding Oso to a Python application
+    to enforce authorization. Start here to understand the basics of using
+    Oso. Then, jump into our more detailed how to section on topics that are
+    important to you.
+aliases:
+  - /getting-started/application/index.html
+---
+
+# Add Oso to Your Python App
+
+In this guide, we'll cover the basics of adding Oso to your Python application
+to enforce authorization. Start here to understand the basics of using
+Oso. Then, jump into our more detailed how to section on topics that are
+important to you.
+
+To use Oso, you'll:
+
+- [Model your authorization policy](model) in Polar using *resources*.
+  Resources allow you to declaratively specify the permissions and roles
+  you want your users to have.
+- [Add authorization enforcement](enforce) throughout your app. Call Oso in
+  your request handlers to reject or accept requests based on your
+  authorization policy.
+
+Depending upon your use case, you may want to:
+
+- [Write Polar rules:](write-rules) Oso can support any authorization model. To
```

Comment:
Do you want to avoid introducing polar on this landing page, or in the add to your app section?

I think a reader should know that Polar is the name of the Oso policy language when they're done reading this section.

Reply:
Landing page

Reply:
I'd introduce it as soon as we ask them to write anything in it/introduce a .polar file.